docs-site/docs/protocol/data-availability.md

```diff
@@ -1,3 +1,47 @@
 ---
+sidebar_position: 1
 title: Data Availability
 ---
+
+Blobs provides reliable and high-throughput Data Availability services. Before diving into exactly what this is, we'll take a quick detour through Rollup scaling solutions.
+
+## Rollup Architecture
+
+The family of Rollup scaling approaches is very broad, and includes systems such as Optimistic Rollups, ZK Rollups, Parachains, and more.
+
+The core idea behind rollups is that new, Layer 2 blockchains can be bootstrapped from mature and secure Layer 1 blockchains simply by posting the transactions of the Layer 2 blockchains onto a Layer 1 chain as raw data. Any node scanning the Layer 1 blocks for Layer 2 transactions, and executing those Layer 2 transactions in order, will achieve the same outcome as any other node which does so. Therefore, the Layer 2 blockchain, or Rollup, is as secure as the Layer 1's transaction ordering guarantees. The terms Layer 2 and Rollup may be used interchangeably.
+
+Rollups are incredibly simple. The difficult part is convincing an agent (in this case, a user, smart contract, or another blockchain) what the state of the rollup is, without requiring that agent to execute every transaction from the rollup in order. Architectural distinctions between rollups emerge based on the **proving protocol** that is used to prove the current state of the rollup to an agent. Rollups may be designed to be compatible with multiple proving protocols simultaneously or may be optimized for a particular type of proving protocol.
+
+All Rollup protocols follow roughly the same blueprint:
+  1. **Sequencing**: Data which encodes one or more transactions is posted to a Layer 1 blockchain and inherits its transaction ordering guarantees. This is the only **required** step for a rollup.
```

Comment:
Not sure what you mean by this is the only required step. So DA is not? Because below you also write that sequencing and data availability are common to all protocols.